### PR TITLE
VIH-10800  refactor video api conferences

### PR DESCRIPTION
**Pull request by will-craig**

### CODE REVIEW

File: VideoApi/VideoApi.Contract/Responses/ConferenceCoreResponse.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    /// </summary>
+    public ConferenceState CurrentStatus { get; set; }
+    
+    /// <summary>
+    /// Is the waiting room still accessible for the conference
+    /// </summary>
+    public bool IsWaitingRoomOpen { get; set; }
+    
+    /// <summary>
+    /// FOR TESTING ONLY: The case name of the conference
```
Comment (shaed-parkar) [inline VideoApi/VideoApi.Contract/Responses/ConferenceCoreResponse.cs:52]: Which testing purpose is this for?

**Review by shaed-parkar — COMMENTED** [review]
Good house keeping. Just a question about the property for testing.

File: VideoApi/VideoApi.Contract/Responses/ConferenceCoreResponse.cs (hunk 1 above)
Comment (will-craig) [reply VideoApi/VideoApi.Contract/Responses/ConferenceCoreResponse.cs:52]: It's used in the tear downs (ac and integration tests), to query all the test conferences on, so they can be removed in the cleanup hook. I couldn't really think of a better alternative, and thought I'd tackle it in another ticket.

**Review by shaed-parkar — APPROVED** [review]

**Review by ronaldmalone — APPROVED** [review]

**Pull request merged**